app/src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -36,25 +43,16 @@ import {
 } from '@/Services/modules/projects'
 import { EventsState, setEvents } from '@/Store/Events'
 import { ProjectsState, setProjects } from '@/Store/Projects'
```

Comment:
It looks like their being used to store data in the Redux store - _"When data has loaded, store data in the Redux store so it can be used by other containers/components"_

Will be honest - have no idea what that means  @joefhall I think you might have added the code for this?

Reply:
Yep they’re being used further down in this file — when the container has loaded projects or events data, it stores it in the Redux store for use by other components e.g. projects search.